modules/mfa/www/prompt-for-mfa.php

```diff
@@ -96,6 +96,25 @@
 
 $globalConfig = Configuration::getInstance();
 
+$otherOptions = array_filter($mfaOptions, function($option) use ($mfaId) {
```

Comment:
Hadn't see that syntax before! anonymous function with a use.

Reply:
I didn't write it, so I can't take any credit or blame.